File: tests/NewCompiler/J2J_NoReturnResults/jt5.c
jasync localyou(int c, char *s) {
    while(1) {
        jsys.sleep(1000000);
        printf("############-->>> Hello YOU -- only local tasks...  %d, %s\n", c, s);
    }
}

int main(int argc, char *argv[])
{
    localyou(10, "cxxxxxxxx");
    return 0;
}

File: tests/NewCompiler/J2J_NoReturnResults/jt5.js
let count = 10;

// @ToasterConfig
// Fogs: 1
// Devices: 1

jasync xyz(hello: char*) {
    console.log("Message from controller ...", hello);
}

async function sleep(n) {
    return new Promise((resolve)=> {
        setTimeout(()=> {
            resolve();
        }, n);
    });
}

if (jsys.type === 'fog') {
	while (1) {
		xyz("hi").catch((e) => console.log(e, " at ", jsys.type));
		await sleep(1000);
	}
}
